packages/platform/platform-express/src/components/PlatformExpress.ts

```
@@ -114,15 +116,30 @@ export class PlatformExpress extends PlatformAdapter<Express.Application> {
 
   mapLayers(layers: PlatformLayer[]) {
     const rawApp: any = this.app.getApp();
+    const v = "v" + version.split(".")[0];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)